integration-tests/src/test/java/oracle/kubernetes/operator/JrfInOperatorTest.java

```diff
@@ -403,6 +412,271 @@ public void testAutoAndCustomSitConfigOverrides() throws Exception {
     logger.info("SUCCESS - " + testMethod);
   }
 
+  /**
```

Comment:
This test class is growing, you may want to group the new use cases you are testing and move them into separate classes.